src/screens/editProfile/components/ProfileDetails.tsx

```diff
@@ -34,6 +34,7 @@ export const ProfileDetails = ({ errors, control, hasValueChanged }: UserData) =
           name="firstName"
           inputLabel={t('userFirstName')}
           placeholder={t('firstNamePlaceholder')}
+          maxLength={20}
```

Comment:
there is commonInputProps obj at the top, maybe change its maxLength to 20 for all input components and remove this prop?